app/main.py

```diff
@@ -2,0 +1,9 @@
+def copy_file(command: str) -> None:
+    command = command.split()
+    print(command)
+    if command[1] == command[2]:
+        pass
```

Comment:
Bug: If the source and destination files are the same, the code does nothing. This might not be the expected behavior. If the source and destination files are the same, you should either handle this case differently or at least inform the user about it.